Cargo.toml

```diff
@@ -74,6 +74,8 @@ alloc = []
 derive = ["zerocopy-derive"]
 simd = []
 simd-nightly = ["simd"]
+# This feature enables support for the unstable f16 and f128 types
```

Comment:
```suggestion
```

Can you instead document the feature in the doc comment at the crate root in `src/lib.rs`?